gui/static/js/components/controlpanel.js

```diff
@@ -29,17 +29,25 @@ export default class ControlPanel extends React.Component {
 
     onKeyDown(event){
         if (this.state.keyboard){
-            if (event.key == 'w'){
-                this.sendMotors(100, 100, 100, 100);
+           if(event.keyCode==87) {
```

Comment:
Would be helpful if you put in comments under the if statements, as such:
```JavaScript
if (event.keyCode == 87) { 
    // If the 'W' key is pressed
    setTimeout() // etc.
} 
```

This allows code to be more readable in the future. Here and below.

Reply:
got it, I would change it today